inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/kafka/src/main/java/org/apache/inlong/sort/kafka/table/UpsertKafkaDynamicTableFactory.java

```diff
@@ -96,6 +102,17 @@ public class UpsertKafkaDynamicTableFactory
             DynamicTableSinkFactory {
 
     public static final String IDENTIFIER = "upsert-kafka-inlong";
+    public static final ConfigOption<String> SINK_MULTIPLE_PARTITION_PATTERN =
+            ConfigOptions.key("sink.multiple.partition-pattern")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "option 'sink.multiple.partition-pattern' used when the partitioner is raw-hash， and used to pass in the primary key partition in primary key partitioners");;
```

Comment:
description need modify

Reply:
done

Reply:
`used to pass in the primary key partition in primary key partitioners` is error